CONTRIBUTING.md

```diff
@@ -0,0 +1,178 @@
+[Prerequisites](#prerequisites).
+
+If you plan to contribute code, it's recommended to set up a virtual environment for development, for example with 
+[conda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment):
+
+```bash
+conda create --name <my-env>
+source activate <my-env>
+```
+
+Now you're ready to make changes to the codebase! Feel free to explore the project and make improvements. Before 
+submitting a pull request, make sure to test your changes thoroughly. Run any relevant tests and ensure that the code 
+meets the project's coding standards, see [Code Style](#code-style). We have used 
+[pylint](https://pypi.org/project/pylint/) and [Flake8](https://flake8.pycqa.org/en/latest/) for code quality improvement.
+
+Once you're happy with your changes, push your branch to GitHub and submit a pull request. Be sure to provide a clear 
+description of the changes you've made and any relevant information for reviewers.
+
+### Issue tracking
+We use GitHub Issues to track bugs, feature requests, and other tasks related to the project. If you encounter any 
+issues or have suggestions for improvements, please open a new issue on the 
+[GitHub repository](https://github.com/wetransform-os/ProMCDA).
+
+Before opening a new issue, search existing issues to see if the problem or feature request has already been reported.
+
+Navigate to the repository of the project. In the repository menu, click on the "Issues" tab and create a new issue.
+Fill out the issue title and description. Be as descriptive as possible, including steps to reproduce the issue 
+if applicable. Provide detailed information about the issue, including what you expected to happen and what actually 
+happened. If applicable, include code snippets, error messages, or steps to reproduce the issue. 
+
+Optionally, you can assign labels to categorize the issue (e.g., new feature, bug, enhancement, documentation).
+We use the following labels to categorize issues:
+
+*Bug*: Indicates a problem with the current implementation that needs to be fixed.
+*Feature Request*: Suggests a new feature or enhancement to be added to the project.
+*Documentation*: Relates to improvements or issues with project documentation.
+
+Click the green "Submit new issue" button to create the issue when you are ready.
+
+### Making changes
+Contributing to the project is easy! Follow these steps to make changes and submit them for review:
+
+#### Fork the repository
+Go to the [GitHub repository of the project] (https://github.com/wetransform-os/ProMCDA) and click the "Fork" button 
+in the top-right corner to create a copy of the repository in your GitHub account. Clone your forked repository from 
+your GitHub account to your local machine using the following command:
+
+```bash
+Copy code
+git clone https://github.com/your-username/project.git
+```
+
+Replace "your-username" with your GitHub username. 
```

Comment:
Also, here you then just proposed to clone the repo but nothing about the requirements and all. I believe this would be another setup where use has to install the requirements too. 

In my opinion this complete block should move to ### Getting Started